TODO.md

```diff
@@ -0,0 +1,6 @@
+- [ ] Only trigger timers in later_posix.cpp and later_win32.cpp if we're at the global loop
+- [ ] User timer in later_win32.cpp is pretty inefficient, have it stop waking so much
+
+## Open questions
+
+* Should private loops have explicit or implicit/automatic extent? i.e. once a private loop is created, is it manually destroyed or is it "garbage collected"? (The latter will need extra work, if native code on background thread needs to schedule work for the private loop--might need handles or explicit reference counting, since cannot rely on R finalizers)
```

Comment:
If everything in TODO.md has been addressed, can you delete it?

Reply:
I think the the Windows timer issue hasn't been addressed. But seems like it should be a separate issue. If you want, I can file an issue for it.

Reply:
Nah I'm fine if you leave it in TODO. Thanks